.structure-config

```diff
@@ -0,0 +1 @@
+INFRASTRUCTURE_FILES='.github/ dockerfile/ scripts/testing/ scripts/githooks/ translation-canary/ Makefile.am'
```

Comment:
`Makefile.am` is included in anaconda srpm (anaconda-34.25.0.7-1.el9), also `translation-canary/xgettext_werror.sh` and the `translation-canary/translation_canary` directory are included there - so I do not think they should be marked as infrastructure files.